repos/system_upgrade/el8toel9/actors/checkarmbootloader/libraries/checkarmbootloader.py

```diff
@@ -0,0 +1,46 @@
+from leapp import reporting
+from leapp.libraries.common.config.architecture import ARCH_ARM64, matches_architecture
+from leapp.libraries.common.config.version import get_source_version, get_target_version
+from leapp.libraries.stdlib import api
+
+
+def _inhibit_upgrade():
+    title = 'Upgrade RHEL {} to RHEL {} not possible for ARM machines.'.format(
+        get_source_version(), get_target_version())
+    summary = ('Due to the incompatibility of the RHEL8 bootloader with a '
+               'newer version of kernel on RHEL9 a direct upgrade '
+               'cannot be performed for versions 9.5 and up.')
```

Comment:
This could be little bit disturbing for users as they will read it as: _upgrades to 9.5 and higher will not be possible on arm_. From this point we want to make it more clear that this is temporary solution but
* we do not want to make a promise neither - we should not regarding RH policies
* and we want to be clear that we do not expect any hotfix for RHEL 9.5 neither

```suggestion
    summary = (
        'Due to the incompatibility of the RHEL 8 bootloader with a newer version of kernel on RHEL {}'
        ' for ARM machines, the direct upgrade cannot be performed to this RHEL'
        ' system version now. The fix is not expected to be delivered during the RHEL 9.5 lifetime.'
        .format(get_target_version())
    )        
```

Reply:
Yes I agree. Fixed!